diff --git a/node_modules/@comapeo/core/src/fastify-plugins/maps.js b/node_modules/@comapeo/core/src/fastify-plugins/maps.js
index 6009b6c..855dee9 100644
--- a/node_modules/@comapeo/core/src/fastify-plugins/maps.js
+++ b/node_modules/@comapeo/core/src/fastify-plugins/maps.js
@@ -87,13 +87,14 @@ export async function plugin(fastify, opts) {
     })
   }
 
-  const fallbackMapReader = new ReaderWatch(opts.fallbackMapPath)
+  // const fallbackMapReader = new ReaderWatch(opts.fallbackMapPath)

-  fastify.addHook('onClose', () => fallbackMapReader.close().catch(noop)) 
+  // fastify.addHook('onClose', () => fallbackMapReader.close().catch(noop))
 
   fastify.register(createServer, {
     prefix: FALLBACK_MAP_PREFIX,
-    reader: fallbackMapReader,
+    // reader: fallbackMapReader,
+    filepath: opts.fallbackMapPath,
   })
 
   fastify.get('/style.json', async (_request, reply) => {
