diff --git a/node_modules/@comapeo/map-server/dist/routes/root.js b/node_modules/@comapeo/map-server/dist/routes/root.js
index e98294d..d8b4886 100644
--- a/node_modules/@comapeo/map-server/dist/routes/root.js
+++ b/node_modules/@comapeo/map-server/dist/routes/root.js
@@ -9,7 +9,7 @@ const MAP_SHARES_BASE = '/mapShares/';
 const DOWNLOADS_BASE = '/downloads/';
 const { preflight } = cors({
     origin: '*',
-    allowMethods: ['GET', 'POST', 'DELETE', 'OPTIONS'],
+    allowMethods: ['GET', 'POST', 'DELETE', 'OPTIONS', 'PUT'],
     allowHeaders: ['Content-Type'],
 });
 /**
