diff --git a/node_modules/@electron-forge/publisher-github/dist/PublisherGithub.js b/node_modules/@electron-forge/publisher-github/dist/PublisherGithub.js
index 128ec61..fd9af08 100644
--- a/node_modules/@electron-forge/publisher-github/dist/PublisherGithub.js
+++ b/node_modules/@electron-forge/publisher-github/dist/PublisherGithub.js
@@ -22,7 +22,10 @@ class PublisherGithub extends publisher_base_1.PublisherBase {
         const { config } = this;
         const perReleaseArtifacts = {};
         for (const makeResult of makeResults) {
-            const release = makeResult.packageJSON.version;
+            // const release = makeResult.packageJSON.version;
+            // NOTE: Our release versions only use the minor and patch versions
+            const [_, minor, patch] = makeResult.packageJSON.version.split('.')
+            const release = `${minor}.${patch}`
             if (!perReleaseArtifacts[release]) {
                 perReleaseArtifacts[release] = [];
             }
