diff --git a/node_modules/@electron-forge/core/dist/util/forge-config.js b/node_modules/@electron-forge/core/dist/util/forge-config.js
index 07b93f4..98ed248 100644
--- a/node_modules/@electron-forge/core/dist/util/forge-config.js
+++ b/node_modules/@electron-forge/core/dist/util/forge-config.js
@@ -163,6 +163,7 @@ exports.default = async (dir) => {
             }
             // The loaded "config" could potentially be a static forge config, ESM module or async function
             const loaded = (await loadFn(forgeConfigPath));
+            console.log("LOADED", loaded)
             const maybeForgeConfig = 'default' in loaded ? loaded.default : loaded;
             forgeConfig = typeof maybeForgeConfig === 'function' ? await maybeForgeConfig() : maybeForgeConfig;
         }
