diff --git a/node_modules/@homebridge/ciao/lib/CiaoService.js b/node_modules/@homebridge/ciao/lib/CiaoService.js
index a70f9b4..6140d2c 100644
--- a/node_modules/@homebridge/ciao/lib/CiaoService.js
+++ b/node_modules/@homebridge/ciao/lib/CiaoService.js
@@ -473,11 +473,11 @@ class CiaoService extends events_1.EventEmitter {
             // thus, when sending the first request, the socket will be bound, and we don't need to wait here
             this.emit("republish" /* InternalServiceEvent.REPUBLISH */, error => {
                 if (error) {
-                    console.log("FATAL Error occurred trying to re-announce service " + this.fqdn + "! We can't recover from this!");
-                    console.log(error.stack);
-                    process.exit(1); // we have a service which should be announced, though we failed to re-announce.
-                    // if this should ever happen in reality, whe might want to introduce a more sophisticated recovery
-                    // for situations where it makes sense
+                    console.error("Error occurred trying to re-announce service " + this.fqdn + ". Service is now unannounced and may need to be re-advertised.");
+                    console.error(error.stack);
+                    if (this.listeners("announce-error").length > 0) {
+                        this.emit("announce-error", error);
+                    }
                 }
             });
         }
diff --git a/node_modules/@homebridge/ciao/lib/Responder.js b/node_modules/@homebridge/ciao/lib/Responder.js
index d690074..d7560c1 100644
--- a/node_modules/@homebridge/ciao/lib/Responder.js
+++ b/node_modules/@homebridge/ciao/lib/Responder.js
@@ -641,11 +641,11 @@ class Responder {
                 // noinspection JSIgnoredPromiseFromCall
                 this.republishService(service, error => {
                     if (error) {
-                        console.log(`FATAL Error occurred trying to resolve conflict for service ${service.getFQDN()}! We can't recover from this!`);
-                        console.log(error.stack);
-                        process.exit(1); // we have a service which should be announced, though we failed to reannounce.
-                        // if this should ever happen in reality, whe might want to introduce a more sophisticated recovery
-                        // for situations where it makes sense
+                        console.error(`Error occurred trying to resolve conflict for service ${service.getFQDN()}. Service is now unannounced and may need to be re-advertised.`);
+                        console.error(error.stack);
+                        if (service.listeners("announce-error").length > 0) {
+                            service.emit("announce-error", error);
+                        }
                     }
                 }, true);
             }
