diff --git a/node_modules/rpc-reflector/lib/types.d.ts b/node_modules/rpc-reflector/lib/types.d.ts
index 3fa8a3e..e89dc7b 100644
--- a/node_modules/rpc-reflector/lib/types.d.ts
+++ b/node_modules/rpc-reflector/lib/types.d.ts
@@ -1,16 +1,10 @@
 /// <reference types="node" />
 /// <reference types="node" />
 /// <reference types="node" />
-import { ErrorObject } from 'serialize-error';
-import { EventEmitter } from 'events';
-import { Readable } from 'stream';
-export declare enum msgType {
-    REQUEST = 0,
-    RESPONSE = 1,
-    ON = 2,
-    OFF = 3,
-    EMIT = 4
-}
+import type { ErrorObject } from 'serialize-error';
+import type { EventEmitter } from 'events';
+import type { Readable } from 'stream';
+import { msgType } from './constants';
 export interface test {
     foo: boolean;
 }
diff --git a/node_modules/rpc-reflector/lib/types.ts b/node_modules/rpc-reflector/lib/types.ts
index d334b37..e49c9dd 100644
--- a/node_modules/rpc-reflector/lib/types.ts
+++ b/node_modules/rpc-reflector/lib/types.ts
@@ -1,14 +1,8 @@
-import { ErrorObject } from 'serialize-error'
-import { EventEmitter } from 'events'
-import { Readable } from 'stream'
+import type { ErrorObject } from 'serialize-error'
+import type { EventEmitter } from 'events'
+import type { Readable } from 'stream'
 
-export enum msgType {
-  REQUEST = 0,
-  RESPONSE = 1,
-  ON = 2,
-  OFF = 3,
-  EMIT = 4,
-}
+import { type msgType } from './constants'
 
 export interface test {
   foo: boolean
