diff --git a/node_modules/styled-map-package/dist/server.js b/node_modules/styled-map-package/dist/server.js
index 377124f..dce27a7 100644
--- a/node_modules/styled-map-package/dist/server.js
+++ b/node_modules/styled-map-package/dist/server.js
@@ -14,10 +14,14 @@ function createServer(fastify, opts, done) {
   if (!("reader" in opts)) {
     fastify.addHook("onClose", () => reader.close().catch(noop));
   }
-  fastify.get("/style.json", async () => {
+  fastify.get("/style.json", async (_req, reply) => {
     try {
       const baseUrl = new URL(fastify.prefix, fastify.listeningOrigin);
       const style = await reader.getStyle(baseUrl.href);
+      // NOTE: Fixes CORS issues while working in development
+      if (process.env.NODE_ENV === 'development') {
+        reply.headers({ 'access-control-allow-origin': '*' })
+      }
       return style;
     } catch (error) {
       if (isFileNotThereError(error)) {
